success-stories/index.html

```diff
@@ -53,6 +53,25 @@ <h3>Testimonials</h3>
     <p>
         Want to share your Spinnaker Success Story? <a href="https://spinnaker.formstack.com/forms/spinnaker_testimonials" target="_blank">Click here!</a>
     </p>
-    
-    <img class="bottom-right-swoosh" src="{{ "assets/images/bottom-right-swoosh.svg" | absolute_url }}" alt="Spinnaker Swoosh"/>
 </div>
+<div class="videos-container">
+    <h3>Videos</h3>
+    <p>Conference talks, interviews, and video testimonials from the Spinnaker Community</p>
+    <div class="videos-wrapper">
+      <h4>Jens Vanderhaeghe of Airbnb on Migrating From Custom Deployment Tooling To Spinnaker</h4>
+    <iframe width="560" height="315" src="https://www.youtube.com/embed/vQCkAkya_Yg" frameborder="0" allow="autoplay; encrypted-media" allowfullscreen></iframe>
```

Comment:
Could we shrink the iframe? They videos seem needlessly gigantic on the page

Reply:
Sure. I agree they look gigantic and several times have checked that they're the same size as the tutorial videos. I think it's just the different formatting on that md page that makes those look less gross? Working on shrinkage now